designer/server/src/test/scala/pl/touk/nussknacker/ui/api/helpers/EspItTest.scala

```diff
@@ -173,24 +173,24 @@ trait EspItTest extends LazyLogging with WithHsqlDbTesting with TestPermissions
       updateProcess(processName, process)(testCode)
     }
 
-  protected def saveProcess(process: DisplayableProcess)(testCode: => Assertion): Assertion =
-    createProcessRequest(ProcessName(process.id)) { code =>
+  protected def saveProcess(process: DisplayableProcess, category: ProcessingType)(testCode: => Assertion): Assertion =
+    createProcessRequest(ProcessName(process.id), category) { code =>
       code shouldBe StatusCodes.Created
       updateProcess(process)(testCode)
     }
 
-  protected def createProcessRequest(processName: ProcessName)(callback: StatusCode => Assertion): Assertion =
-    Post(s"/processes/${processName.value}/$TestCat?isSubprocess=false") ~> processesRouteWithAllPermissions ~> check {
+  protected def createProcessRequest(processName: ProcessName, category: String = TestCat)(callback: StatusCode => Assertion): Assertion =
+    Post(s"/processes/${processName.value}/$category?isSubprocess=false") ~> processesRouteWithAllPermissions ~> check {
       callback(status)
     }
 
-  protected def saveSubProcess(process: CanonicalProcess)(testCode: => Assertion): Assertion = {
-    val displayable = ProcessConverter.toDisplayable(process, TestProcessingTypes.Streaming)
+  protected def saveSubProcess(process: CanonicalProcess, category: String)(testCode: => Assertion): Assertion = {
+    val displayable = ProcessConverter.toDisplayable(process, TestProcessingTypes.Streaming, category)
```

Comment:
I wonder, we assume fixed ProcessingType here, maybe we can do so also for category?